src/Draco.Editor.Web/Program.cs

```diff
@@ -6,30 +6,20 @@
 using Draco.Compiler.Api.Syntax;
 using System.Text.Json;
 using System.Reflection;
+using System.IO;
 
 namespace Draco.Editor.Web;
 
 public partial class Program
 {
     private static string code = null!;
-    private static string selectedOutputType = null!;
 
     public static void Main() => Interop.Messages += OnMessage;
 
     public static void OnMessage(string type, string payload)
     {
         switch (type)
```

Comment:
Do we still need the switch and message type?

Reply:
We only handle one case.

Reply:
No, but I will probably add more messages types later.